src/renderer/screens/manager/AppsList/AppsList.js

```diff
@@ -78,9 +78,16 @@ const AppsList = ({ deviceInfo, state, dispatch, plan, isIncomplete, progress =
     sort,
   );
 
-  const appsInstalling = installQueue.map(name => appByName[name]);
+  /** combined installed apps and installQueue apps with removed duplicates */
+  const combinedInstalledApps = useMemo(
+    () =>
+      [...installQueue.map(name => appByName[name]), ...installedAppList].filter(
+        ({ name }, i, arr) => arr.findIndex(a => a.name === name) === i,
```

Comment:
- not that critical but the complexity of this is not that great (`O(n*n)`?)
- let's discuss why we need that array and if we couldn't offer it in live-common side.

Reply:
I wanted to do same as in mobile, putting current installing apps with installed apps inside the installed section.
As pointed out by arnaud we have to remove duplicates in case installed apps are being reinstalled ie: updates
- I agree this should be done in the live-common part thus removing some issues on requesting installs on dependent apps having their deps currently scheduled but not yet installed.